cypress-io/cypress
# fix(unify): fixing icons and supporting router within the shared cypress commands package

### Summary

Currently, the component tests for the App aren't running and the icons don't look right within the tests. This PR fixes the local component tests for the App.

### How to test

1.  `git checkout fix/icon-colors-and-shared-plugins`
2.  `cd packages/app`
3.  `yarn cypress:open` and the component tests should render and look correct.

### Changes

1.  The current frontend-shared commands for Cypress are importing application code. This causes issues when one project uses dependencies that another project does not. This PR refactors `frontend-shared/commands/command.ts` to export a factory function where each project can inject the plugins it needs.
2.  The component tests were broken when adding `CYPRESS_INTERNAL_CLOUD_ENV` during the remote schema stitching PR.
3.  The icons (mainly the settings cog) weren't getting colored correctly.
